website/routes.py

```diff
@@ -53,11 +53,24 @@
 from website.ember_osf_web.decorators import ember_flag_is_active
 
 
+def set_status_message(user):
+    if user and not user.accepted_terms_of_service:
+        status.push_status_message(
+            message=language.TERMS_OF_SERVICE.format(settings.API_DOMAIN, user._id),
+            kind='default',
+            dismissible=True,
```

Comment:
Should this be `False`, since it's not actually dismissible until the user accepts the TOS?